### PR TITLE
Update AgentQnA example for v1.1 release

### PR DESCRIPTION
**Pull request by minmin-intel**
## Description

1. Use rag_agent-llama and react_llama, which are developed for llama3.1-70b-instruct served with tgi-gaudi
2. update retrieval tool given the changes in RetrievalToolGateWay

## Issues

https://github.com/opea-project/GenAIExamples/issues/718

## Type of change
- [x] Others (enhancement, documentation, validation, etc.)

## Dependencies

No newly introduced 3rd party dependency if exists.

## Tests

Unit test in tests folder

### CODE REVIEW

**Review by lvliang-intel — APPROVED** [review]

File: AgentQnA/tests/4_launch_and_validate_agent_tgi.sh
Comment (lvliang-intel) [inline AgentQnA/tests/4_launch_and_validate_agent_tgi.sh:1]: The first character of a file name should not be a digit for better naming practices.

**Comment by lkk12014402** [comment]
hi minmin, we should update readme for the usage.

File: AgentQnA/tests/4_launch_and_validate_agent_tgi.sh
Comment (minmin-intel) [reply AgentQnA/tests/4_launch_and_validate_agent_tgi.sh:1]: Updated the filenames.

**Comment by minmin-intel** [comment]
> hi minmin, we should update readme for the usage.

@lkk12014402 Please review README. I put everything in the one README in the root folder. I added a very short README in the xeon folder explaining that we need to use OpenAI for LLM on Xeon and deploy retrieval tool megaservice on xeon.

**Review by lkk12014402 — APPROVED** [review]

**Pull request merged**